pkg/service/test/test.go

```diff
@@ -274,7 +275,25 @@ func (t *tester) Test(path string, testReportPath string, generateTestReport boo
 		testSuiteNames = append(testSuiteNames, testSuiteName)
 	}
 
-	sort.Strings(testSuiteNames)
+	sort.SliceStable(testSuiteNames, func(i, j int) bool {
+
+		test1 := strings.Split(testSuiteNames[i], "-")
+		test2 := strings.Split(testSuiteNames[j], "-")
```

Comment:
rather than test1 ~ testSuitePartsI and testSuitePartsJ can be a better. similarly instead of num1 something more specific testSuiteIDNumberI and testSuiteIDNumberJ can be used a better explainer.

Reply:
ok